add: strings (#20)
Wiki-Knights/CS-Materials: src/intro-to-c/strings/strings-practice.md

# Strings Practice

> 

Reviewer: Add from the answers document
```suggestion

A fun thing to do in these exercises is to create a separate function and call it in main. That way, you can have your own library of functions and reuse it if you need.
```

> 1. Write a function that receives a word and return its length without using any libraries.
> 
> 2. Write a function that receives a word and turns it into uppercase.
> 
> 3. Write a function that receives two words and checks if they are equal.
> 
> 4. Write a function that receives a word and check if it is a palindrome.
> 
> 5. Write a function that receives a word and reverses it.

Reviewer: Replacing "word" with "string" so it is more generic
```suggestion
1. Write a function that receives a string and return its length without using any libraries.

2. Write a function that receives a string and turns it into uppercase.

3. Write a function that receives two strings and checks if they are equal.

4. Write a function that receives a string and check if it is a palindrome.

5. Write a function that receives a string and reverses it.
```

> 6. **CHALLENGE** How would you read a first and last name and display it to the screen?

Reviewer: Reword because I was super confused at first what this meant
```suggestion
6. **CHALLENGE:** How would you read a first and last name from the terminal, save it to a string, and display it to the screen?
```
